CHANGELOG.md

```diff
@@ -40,6 +40,20 @@ Current
 - [Filter Code now intersects security constraints instead of unioning with requests](https://github.com/yahoo/fili/issues/812)
     * Switched to ensure security and request filters don't merge but instead intersect
 
+- [Bump Jackson version to patch vulnerability](https://github.com/yahoo/fili/issues/770)
+    * Bumped dependency version to 2.9.5 
+    * Made serialization order more specific in several classes
+    * Fixed bad format in error message
+    * Moved tests off of serialization of `SimplifiedIntervalList`.  That's turning out to be hard to solve.
+
+- [Bump lucene version to patch vulnerability](https://github.com/yahoo/fili/issues/819)
```

Comment:
I feel like we should explicitly call out that this forces a breaking change on Pagination, since anybody who has their own search providers has to work with them. In particular that the number of results, and the last page are now (or should be) longs.

Reply:
Reverting the breaking change in favor of an error on largeints and no interface changes.